Gemfile

```diff
@@ -5,7 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-gem 'conjur-api', '~> 5.0.0'
+gem 'conjur-api', '~> 5.1'
```

Comment:
I usually prefer to only accept patch updates automatically just for safety's sake, but perhaps since this gem is under our control it's a little less risky.

Reply:
Updated.